superset/config.py

```diff
@@ -382,11 +382,29 @@ def _try_json_readsha(  # pylint: disable=unused-argument
 #          ['#006699', '#009DD9', '#5AAA46', '#44AAAA', '#DDAA77', '#7799BB', '#88AA77',
 #          '#552288', '#5AAA46', '#CC7788', '#EEDD55', '#9977BB', '#BBAA44', '#DDCCDD']
 #     }]
-#
 
 # This is merely a default
 EXTRA_CATEGORICAL_COLOR_SCHEMES: List[Dict[str, Any]] = []
 
+# CUSTOM_THEME is used for adding custom theme to superset
+# example code for "My theme" custom scheme
+# CUSTOM_THEME = {
+#   "borderRadius": 4,
+#   "colors": {
+#     "primary": {
+#       "base": 'red',
+#     },
+#     "secondary": {
+#       "base": 'green',
+#     },
+#     "grayscale": {
+#       "base": 'orange',
+#     }
+#   }
+# }
+
+CUSTOM_THEME = {}
```

Comment:
I'd prefer to call this `THEME_OVERRIDES` to highlight that it gets merged into the theme.

Reply:
done